sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_async.py

```diff
@@ -172,243 +164,19 @@ async def test_all_successful_passing_dict_pii_entities_task(self, client):
                     self.assertIsNotNone(entity.offset)
                     self.assertIsNotNone(entity.confidence_score)
 
-    @pytest.mark.playback_test_only
-    @GlobalTextAnalyticsAccountPreparer()
```

Comment:
you'll notice some tests are deleted. This is because we have quite a few redundant tests, and since analyze tests take so long, I wanted to remove the redundant ones to make room for incoming tests